magefile.go

```diff
@@ -201,6 +208,18 @@ func customizePackaging() {
 			args.Spec.ReplaceFile("/usr/share/{{.BeatName}}/README.md", readmeTemplate)
 			args.Spec.Files["/usr/share/{{.BeatName}}/"+ingestTarget] = ingest
 
+			// update config file Owner
+			pf := args.Spec.Files["/etc/{{.BeatName}}/{{.BeatName}}.yml"]
+			pf.Owner = mage.BeatUser
+			args.Spec.Files["/etc/{{.BeatName}}/{{.BeatName}}.yml"] = pf
```

Comment:
`fields.yml` is still owned by root, that is not a problem, right? but maybe would be better that the `etc/apm-server` is also owned by `apm-server` user?

Reply:
good call, will investigate whether that's an issue.

Reply:
I wasn't able to trigger any issues with this, even the pipeline registration continues to work with the current permissions.  For comparison, elasticsearch leaves `/etc/elasticsearch` as root user owned, elasticsearch group owned with sticky bit.  I don't think we need to do that but could follow up with that separately if we do find it's necessary after beta testers get their hands on this.